manim/mobject/mobject.py

```diff
@@ -69,6 +72,37 @@ def generate_points(self):
         pass
 
     def add(self, *mobjects):
+        """Add mobjects as submobjects.
+
+        The mobjects are added to self.submobjects.
+
+        Parameters
+        ----------
+        mobjects : List[:class:`Mobject`]
```

Comment:
we aren't expecting lists of mobjects, right?
```suggestion
        mobjects : :class:`Mobject`
```

Reply:
`mobjects` is always a list, no? How to specify it is packing all arguments passed?